docker/Dockerfile.production

```diff
@@ -0,0 +1,23 @@
+# Build
```

Comment:
Couldn't you use the same Dockerfile?

Reply:
Nops! One intends to start a dev frontend, and the other to build it for production and serve using Nginx. Feel free to suggest an alternative.

Reply:
Actually, now talking to @rluders I understood why.
I will put this as resolved.